session/dto.go

```diff
@@ -24,10 +24,10 @@ type ID string
 
 // Session structure holds all required information about current session between service consumer and provider
 type Session struct {
-	ID              ID
-	Config          ServiceConfiguration
-	ConsumerID      identity.Identity
-	DestroyCallback DestroyCallback
+	ID         ID
+	Config     ServiceConfiguration
+	ConsumerID identity.Identity
+	Stop       chan struct{}
```

Comment:
Naming. it's not a channel to stop something, it's a channel to notify. Better name would be "Done", "Gone", "Closed" or smth like that.

Reply:
Renamed.